roles/openshift_management/files/examples/container_providers.yml

```diff
@@ -0,0 +1,22 @@
+---
+container_providers:
+  - connection_configurations:
+      - authentication: {auth_key: "management-admin-token-here", authtype: bearer, type: AuthToken}
+        endpoint: {role: default, security_protocol: ssl-without-validation, verify_ssl: 0}
+    hostname: "OCP/Origin cluster hostname (providing API access)"
+    name: openshift-management
+    port: 8443
+    type: "ManageIQ::Providers::Openshift::ContainerManager"
+# Copy and update for as many OCP or Origin providers as you want to
+# add to your management service
+  # - connection_configurations:
+  #     - authentication: {auth_key: "management-admin-token-here", authtype: bearer, type: AuthToken}
+  #       endpoint: {role: default, security_protocol: ssl-without-validation, verify_ssl: 0}
+  #   hostname: "OCP/Origin cluster hostname (providing API access)"
+  #   name: openshift-management
+  #   port: 8443
+  #   type: "ManageIQ::Providers::Openshift::ContainerManager"
+management_server:
+  hostname: "Management server hostname (providing API access)"
+  user: admin
+  password: smartvm
```

Comment:
Should these be using the variables or should they always be admin/smartvm? Are the admins expected to update it here?

Reply:
Those are the defaults. Since the instructions require you to edit that file anyway you have the opportunity to change those values as necessary. Also, that file is *specifically* for the case where a customer is trying to add *arbitrary* OCP instances to *any* arbitrary MIQ/CFME instance. That's the reason it isn't variablized.